cluster/ubuntu/util.sh

```diff
@@ -515,21 +661,35 @@ function push-node() {
   fi
 
   export KUBECTL_PATH="${KUBE_ROOT}/cluster/ubuntu/binaries/kubectl"
-  local node_ip=${1}
-
+  
   setClusterInfo
-
+  
+  local node_ip=${1}
   local ii=0
   local localexisting=false
```

Comment:
The variable localexisting is un-referenced in this file.  Do we need to assign to it?